bentoml/service/inference_api.py

```diff
@@ -310,6 +312,9 @@ def handle_batch_request(self, requests: Sequence[HTTPRequest]):
             responses = tuple(map(self.output_adapter.to_http_response, results))
             for inf_task, response in zip(inf_tasks, responses):
                 response.headers['X-Request-Id'] = inf_task.task_id
+                if self.output_adapter.cors:
```

Comment:
We will need to change the default value of `cors` parameter to `None` in order to avoid breaking change for our users right?

Reply:
currently the default value is `*`, changing it to `None` would mean users using javascript to visit Lambda endpoint will face a breaking change and will need to specify something like `output=JsonOutput(cors="*")` in their code. But I guess that's a smaller impact than changing the response header to `*` for all BentoML users. what do you think?

Reply:
Good point! I'm not sure why the default value is `*` for lambda. I agree the default value should be `None`.